src/controllers/userProfile.js

```diff
@@ -23,4 +23,15 @@ export default class changeUserProfile {
         response.status(200).json({ status: 200, user: returnProfile(user) });
       }).catch((dbError) => response.status(500).json({ error: 'Database issues', dbError }));
   }
+
+  static async rememberMe(req, res) {
+    try{
+   const userInfo = await models.User.findOne({ where: { id: req.user.id } })
+   const {password, ...rest} = userInfo.dataValues;
+   req.session.profileInfo = rest
+    return res.status(200).send({status:200, message:'Your profile information was saved for next use', data:req.session.profileInfo})
```

Comment:
please change the responce message to make it more understandable.

Reply:
I have reduced the expiration date